r/tools/distro-map.csv

```diff
@@ -0,0 +1,22 @@
+actual,use_this
+fedora-28,ubuntu-18.04
+fedora-29,ubuntu-18.04
+fedora-30,ubuntu-18.04
+fedora-31,ubuntu-18.04
+fedora-32,ubuntu-18.04
+fedora-33,ubuntu-18.04
+fedora-34,ubuntu-18.04
+centos-8,ubuntu-18.04
+rhel-8,ubuntu-18.04
+fedora-19,centos-7
+fedora-20,centos-7
+rhel-7,centos-7
+debian-10,ubuntu-18.04
+debian-11,ubuntu-18.04
+ubuntu-18.10,ubuntu-18.04
+ubuntu-19.04,ubuntu-18.04
+ubuntu-19.10,ubuntu-18.04
+ubuntu-20.04,ubuntu-18.04
+ubuntu-20.10,ubuntu-18.04
+ubuntu-21.04,ubuntu-18.04
+ubuntu-21.10,ubuntu-18.04

```

Comment:
This is missing https://github.com/ursa-labs/arrow-r-nightly/commit/bda16283a911d680c5a0481c39cebeb4379dc57d (see also https://github.com/apache/arrow/issues/13211)